### PR TITLE
Add "Running your first test plan" tutorial

### PR DESCRIPTION
**Pull request by pieqq**
## Description

New section of the base tutorial: Running your first test plan.

This comes after installing Checkbox. In this tutorial we run it, get familiar with the user interface (how to select test plans, jobs, how to process manual or semi-automated jobs, how to re-run jobs) and generate some submissions.

**Note:** This section relies on the latest version of the Tutorial provider developed in a separate PR (#645).

## Resolved issues

CHECKBOX-749

## Tests

Tested locally by running the usual suspects:

- `make run` to get a live preview (you need to go visit http://127.0.0.1:8000/tutorial/using-checkbox/basic-running/index.html to see the page in action)
- `make spelling`, `make linkcheck`

### CODE REVIEW

File: docs/tutorial/using-checkbox/basic-running.rst [hunk 1]
```diff
@@ -0,0 +1,394 @@
...
+    Category: com.canonical.certification::tutorial
+    ... 8< -------------------------------------------------------------------------
+    This job fails!
+    ------------------------------------------------------------------------- >8 ---
+    Outcome: job failed
+    (...)
+
+For each job, we can see:
+
+- How many jobs have run (2), and how many jobs there are in total (11)
```
Comment (tang-mm) [inline docs/tutorial/using-checkbox/basic-running.rst:131]: How about "The number of jobs that have run (2) and the total number of jobs scheduled (11)"?

File: docs/tutorial/using-checkbox/basic-running.rst [hunk 2]
```diff
@@ -0,0 +1,394 @@
...
+- How much estimated time is remaining before the test session is finished (3 minutes)
```
Comment (tang-mm) [inline docs/tutorial/using-checkbox/basic-running.rst:132]: How about "The estimated remaining time before the test session is finished (3 minutes)"

**Review by tang-mm — COMMENTED** [review]
This tutorial session is well curated for beginners, good job! 

Since this page is relatively long, I'd suggest adding a few section headings for easier navigation. For example: "Filtering test plans" "Executing test plans" "Verifying manual jobs" "Checking test reports"

File: docs/tutorial/using-checkbox/basic-running.rst (hunk 2 above)
Comment (pieqq) [reply docs/tutorial/using-checkbox/basic-running.rst]: Fixed in https://github.com/canonical/checkbox/pull/705/commits/171502bdca37d5a4044eaf85ef5d19ac5526a3b8

Thanks!

File: docs/tutorial/using-checkbox/basic-running.rst (hunk 1 above)
Comment (pieqq) [reply docs/tutorial/using-checkbox/basic-running.rst]: Fixed in https://github.com/canonical/checkbox/pull/705/commits/171502bdca37d5a4044eaf85ef5d19ac5526a3b8. Thanks!

**Review by tang-mm — APPROVED** [review]
Looks good to me!

**Pull request merged**